packages/components/src/Button/button.stories.tsx

```diff
@@ -37,6 +37,8 @@ type Args = React.ComponentProps<typeof Button>;
 
 const Template: Story<Args> = (args) => <Button {...args} />;
 
+// TODO: Consolidate with Link stories.
```

Comment:
Since Link and Button are separate components, should we skip consolidating them? It seems reasonable to me to keep their stories completely separate.

Reply:
We definitely don't need to (and it would be less work so ayyyy 👍 👍 ), but I figure it makes sense since they can create the exact same styles and the separation is more related to their underlying element than the appearance. I have a general concern about devs thinking you use `Link` if it looks like a link and `Button` if it looks like a button.

Reply:
>  I have a general concern about devs thinking you use Link if it looks like a link and Button if it looks like a button.

Legit concern 👍 . We'll have to educate and document (which you've already done) that you use Link to get a link, and use Button to get a button, regardless of styling.

To me, consolidating stories seems like it would increase confusion, not decrease. So I'm thinking separate makes the most sense.

Plus the shared styles via Clickable is more of an implementation detail, not inherent to the API, I think.

Reply:
I'll go ahead and remove the todo comments 👍